bindings/python/benchmarks/benchmarks.py

```diff
@@ -280,6 +281,43 @@ def setup_cache(self):
         )
 
 
+class ProfileReadExtensionSmall(Read):
+    schema = Schema({"x": Decimal128Type(), "y": BinaryType(10)})
+    dtypes = np.dtype(np.dtype([("x", np.object_), ("y", np.object_)]))
+
+    def setup_cache(self):
+        coll = db.benchmark
+        coll.drop()
+        base_dict = collections.OrderedDict(
```

Comment:
Why use OrderedDict?

Reply:
It was used in the other tests, switched to dict.